old/model/Field.js

```diff
@@ -0,0 +1,13 @@
+var util = require('../_util');
+
+/**
+ * Stores information about a field in a {@link Model}.
+ */
+function Field(opts) {
```

Comment:
None of the stuff in old/ is being used right now; I wouldn't be surprised if it was internally inconsistent as well